### ISSUE TITLE
quantization-aware training

### ISSUE THREAD

**Issue by jediblu** [pytorch/pytorch#30125]
## To Reproduce

1. git clone https://github.com/pytorch/tutorials.git
1. Build the tutorials following the documentation on https://github.com/pytorch/tutorials
1. Run the tutorials/advanced_source/static_quantization_tutorial.py 

## Expected behavior

There is a comment in static_quantization_tutorial.py that says "quantization-aware training yields an accuracy of over 71% on the entire imagenet dataset, which is close to the floating point accuracy of 71.9%."

We used the entire imagenet dataset to get the accuracy. Accuracy of the fp32 model was 71.86%, as stated. However, the quantization-aware training quantized model was 67.06%, not the 71% stated in the comments.

## Environment

Please copy and paste the output from our
[environment collection script](https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py)
(or fill out the checklist below manually).

You can get the script and run it with:
```
wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
# For security purposes, please check the contents of collect_env.py before running it.
python collect_env.py
```

PyTorch version: 1.3.1
Is debug build: No
CUDA used to build PyTorch: 10.1.243

OS: Ubuntu 18.04.3 LTS
GCC version: (Ubuntu 7.4.0-1ubuntu1~18.04.1) 7.4.0
CMake version: version 3.15.4

Python version: 3.6
Is CUDA available: No
CUDA runtime version: No CUDA
GPU models and configuration: No CUDA
Nvidia driver version: No CUDA
cuDNN version: No CUDA

Versions of relevant libraries:
[pip3] numpy==1.17.4
[pip3] pytorch-sphinx-theme==0.0.24
[pip3] torch==1.3.1
[pip3] torchaudio==0.3.1
[pip3] torchtext==0.4.0
[pip3] torchvision==0.4.2
[conda] Could not collect

## Additional context

cc @jerryzh168 @jianyuh @dzhulgakov @raghuramank100 @jamesr66a

**Comment by raghuramank100** [comment]
Can you provide more details? The tutorial was using a part of imagenet for illustration purposes. The script that we actually used to train mobilenet is now open sourced as part of torchvision at:
https://github.com/pytorch/vision/blob/master/references/classification/train_quantization.py

The quantized mobilenet model (trained using this script) is also available as a pretrained quantized model: https://github.com/pytorch/vision/blob/master/torchvision/models/quantization/mobilenet.py#L12

**Comment by robotcator** [comment]
In https://github.com/pytorch/vision/blob/master/references/classification/ says that 'For Mobilenet-v2, the model was trained with quantization aware training converges at about 10 epochs'. Did you train it on multi-GPU? if so, how to deal with the MovingAverageMinMaxObserver?

**Comment by vkuzo** [comment]
> Did you train it on multi-GPU?

for the original blog post, the QAT part of the training was done using a single GPU.  We've since added support for multi-GPU QAT.  The accuracy of 71%+ on ImageNet1k + MobileNetV2 is reproducible using the script here (https://github.com/pytorch/vision/blob/master/references/classification/train_quantization.py), as @raghuramank100  mentioned.

I'm closing this since we verified reproducibility of the blog post results, @robotcator  I hope this helps and please reopen if I closed this in error!
